main.go

```diff
@@ -62,6 +63,9 @@ func processFile(filename string, in io.Reader, out io.Writer) error {
 	if *underlineHeadings {
 		opts = append(opts, markdown.WithUnderlineHeadings())
 	}
+	if *hardWraps {
+		opts = append(opts, markdown.WithHardWraps())
```

Comment:
I think another 3rd option here would be to automatically change it to X chars wide. We can add that later, but wonder if this option would conflict. e.g if someone will use `WithHardWraps` and `WithMaxLineWidth(80)`

Reply:
Yes, it can be added later! I think a check might be needed for the conflict between them. Also, word wrapping needs to be implemented for this in a way(something like [this](https://pkg.go.dev/github.com/mitchellh/go-wordwrap)) so that there are no formatting surprises! 🙂